main_pullrequest.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/go-github/v28/github"
+
+	clientgithub "github.com/mendersoftware/integration-test-runner/client/github"
+)
+
+func processGitHubPullRequest(ctx *gin.Context, pr *github.PullRequestEvent, githubClient clientgithub.Client, conf *config) error {
+	log := getCustomLoggerFromContext(ctx)
+
+	// Do not run if the PR is a draft
+	if pr.GetPullRequest().GetDraft() {
+		log.Infof("The PR: %s/%d is a draft. Do not run tests", pr.GetRepo().GetName(), pr.GetNumber())
+		return nil
+	}
+
+	if isDependabotPR, err := maybeVendorDependabotPR(log, pr, conf); isDependabotPR {
+		if err != nil {
+			log.Errorf("maybeVendorDependabotPR: %v", err)
+		}
+		return err
```

Comment:
suggestion:
Just be explicit and return `nil`

Reply:
fixed